language/compiler/src/unit_tests/testutils.rs

```diff
@@ -35,7 +35,7 @@ fn compile_script_string_impl(
     code: &str,
     deps: Vec<CompiledModule>,
 ) -> Result<(CompiledScript, Vec<VMStatus>)> {
-    let parsed_program = parse_program(code).unwrap();
```

Comment:
I'd actually recommend leaving the unwrap() calls here - in unit tests, it is easier to deal with a panic at the source than a Result() that gets passed back up the stack.